### ISSUE TITLE
[Documentation] v-dialog tag in documentation is at the wrong spot

### ISSUE THREAD

**Issue by anjakDev** [vuetifyjs/vuetify#11888]
### Environment

**Browsers:** Safari 13.1.1
**OS:** Mac OS 10.15.5

### Steps to reproduce
I've noticed that the source code displayed for the different dialog types always places the v-dialog tag *above* the button component that should be used to activate the dialog (see https://github.com/vuetifyjs/vuetify/blob/3acc307d52494fb67ff1d43b4868ce475ac3476d/packages/docs/src/examples/dialogs/simple/modal.vue#L3 for example). But if you use it like that, it will hide the button if the v-model="dialog" is set to false (i.e. the dialog should not be shown). In the linked code-pen of the documentation, it is used correctly, only surrounding the actual content that should be in the dialog, but not the button triggering the visibility boolean. So I think it would be good to fix it in the source code snippets shown on the website, because it took me a while to figure out what was happening to my button after copying the code for this. 

### Expected Behavior
Copying the source code snippet on the docu page results in a functioning example of triggering a dialog on a button click. 

### Actual Behavior
Copying the source code snippet on the docu page results in a hidden button that doesn't allow any interaction to even trigger the dialog, because the order of the html tags is wrong. 

### Reproduction Link
<a href="https://vuetifyjs.com/en/components/dialogs/#modal" target="_blank">https://vuetifyjs.com/en/components/dialogs/#modal</a>

**Comment by jacekkarczmarczyk** [comment]
Please elaborate
<https://codepen.io/jkarczm/pen/qBbQzpW?editable=true&editors=101%3Dhttps%3A%2F%2Fvuetifyjs.com%2Fen%2Fcomponents%2Fdialogs%2F>

**Comment by anjakDev** [comment]
Right, apparently it was too late and my brain not functional when I checked this, but I can see now that it is in fact the same way in the code-pen. I think I mistook the example without the activator as the one from the modal. I just double-checked with my code, and realised it would hide my button when surrounding it with the v-dialog tag only because I am also using a v-tooltip around the button. Commenting out the v-tooltip fixes this issue. So I can see I made a mistake here. Sorry!
